### PR TITLE
Add Item_Device* searchable lists

### PR DESCRIPTION
**Pull request by cedric-anne**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

### CODE REVIEW

File: inc/item_devices.class.php [hunk 1]
```diff
@@ -105,9 +105,41 @@ function getForbiddenStandardMassiveAction() {
       return $forbidden;
    }
 
-
    function rawSearchOptions() {
-      $tab = parent::rawSearchOptions();
+      $tab = [];
```
Comment (trasher) [inline inc/item_devices.class.php]: Is it expected you no longer call parent method?

**Comment by trasher** [comment]
You have a small CS issue to fix

File: inc/item_devices.class.php (hunk 1 above)
Comment (cedric-anne) [reply inc/item_devices.class.php]: Yes parent provides search options that does not works for this case.

Comment (trasher) [reply inc/item_devices.class.php]: OK :)

**Pull request merged**